cypress/integration/EnrollmentAddEventPage/EnrollmentAddEventPageNavigation/index.js

```diff
@@ -18,3 +18,10 @@ And(/^you see the widget header (.*)$/, (name) => {
             cy.contains(name).should('exist');
         });
 });
+
+And('you see the add event form details', () => {
+    cy.get('[data-test="add-event-enrollment-page-content"]')
+        .within(() => {
+            cy.get('[data-test="edit-event-form"]').should('exist');
```

Comment:
Hey, 
Can you add more strict checks for this feature? I think that besides the form id we can also check that some specific elements inside the form are being displayed.
Thank you!